code/game/mecha/mecha_parts.dm

```diff
@@ -234,12 +234,10 @@
 
 /obj/item/mecha_parts/chassis/phazon/attackby(obj/item/I, mob/user, params)
```

Comment:
You can drop the entire proc override since all it does is can and return the super de
efinition now

Reply:
Removed